packages/mask/src/extension/popups/pages/Wallet/AddToken/index.tsx

```diff
@@ -106,15 +109,33 @@ const SupportedChains = [
     ChainId.xDai,
 ]
 
+const TokenSupportedChains: Web3Helper.ChainIdAll[] = [
```

Comment:
I'd suggest this

```ts
const TokenSupportedChains: Record<Web3Helper.ChainIdAll, boolean> = {
```

Reply:
Why 🤔

Reply:
Less error prone.

Reply:
if you're missing a new chain id, it will be a type error